packages/core/src/utils/getDefaultRequestOptions.ts

```diff
@@ -0,0 +1,21 @@
+import { getHubApiUrl, IHubRequestOptions } from "@esri/hub-common";
+
+/**
+ * Return a minimal IHubRequestOptions object for
+ * use in functions that don't require authentication
+ * but do need some url information
+ *
+ * @export
+ */
+
+export function getDefaultRequestOptions(
+  portalUrl: string = "https://maps.arcgis.com/sharing/rest"
```

Comment:
Should `portalUrl` be `portalUrl?` since we're assigning a default value? `portalUrl?: string = "https://maps.arcgis.com/sharing/rest"`

Also, update method doc w/ args?

Reply:
TS complains (for me anyhow) if you use `?` and assign a default value...

Reply:
![image](https://user-images.githubusercontent.com/119129/118694230-172bfb80-b7c9-11eb-95ca-697cf2be1f5a.png)